### ISSUE TITLE
[Bug] Uninstall script is not wiping clean

### ISSUE THREAD

**Issue by AuthorShin** [IceWhaleTech/CasaOS#714]
Hello
When I install the CasaOS on a fresh new system it will install docker and a lot of other dependencies as well as opening a lot of ports for SMB and etc, but when I uninstall the CasaOS most of them are left over which I don't want anymore.
So that's why I would really appreciate an option when running casaos-uninstall to ask me if I want to delete all those dependencies and close those ports which where opened by CasaOS and it's dependencies. An option like Totally uninstall would be great to bring back the system to it's pre-casaos-install conditions.

**Comment by jerrykuku** [comment]
Yes, I agree with you, I will try to remove the dependencies when uninstall casaos. But so far I haven't found a way to remove docker completely. We can discuss it.

**Comment by tigerinus** [comment]
```bash
docker ps --all --filter label=casaos --format "{{ .Names }}"
```

@jerrykuku ^^^

**Comment by ETWang1991** [comment]
This is really the part that CasaOS doesn't do well at the moment, and we'll fix it as soon as possible.
